src/api.rs

```diff
@@ -29,7 +29,7 @@ pub async fn run_server() -> Result<(), ServerError> {
         .unwrap();
 
     let handle = server.start(module)?;
-    info!("api started at 127.0.0.1:9123");
+    info!(addr = "127.0.0.1:9123", "api started");
```

Comment:
This should be configurable at some point. It's pretty easy to do, see signup-sequencer for example. (Or kzg-ceremony which uses Axum, something we'll want to migrate to)

Reply:
fixed here: https://github.com/worldcoin/tx-sitter/pull/21